pandas/core/internals/managers.py

```diff
@@ -95,9 +95,12 @@ class BlockManager(PandasObject):
     __slots__ = ['axes', 'blocks', '_ndim', '_shape', '_known_consolidated',
                  '_is_consolidated', '_blknos', '_blklocs']
 
-    def __init__(self, blocks, axes, do_integrity_check=True):
+    def __init__(self,
+                 blocks: Sequence[Block],
+                 axes: Sequence[Index],
+                 do_integrity_check: bool = True):
         self.axes = [ensure_index(ax) for ax in axes]
-        self.blocks = tuple(blocks)
+        self.blocks = tuple(blocks)  # type: Tuple[Block, ...]
```

Comment:
can we not use the assignment form of this? e.g.
```self.blocks: Tuple[Block, ...] = .....```

Reply:
Unfortunately not until Py36 (only valid in function signature in Py35)